spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -433,6 +433,42 @@ at the broker by configuring a binding for the outbound target named `error`. Fo
 publish error messages to a broker destination named "myErrors", provide the following property:
 `spring.cloud.stream.bindings.error.destination=myErrors`
 
+===== Using @ErrorListener for handling error messages
+
+The error messages sent to the global Spring Integration `errorChannel` can be consumed via explicit methods annotated with `@ErrorListener`.
+Since the `errorChannel` is a pub-sub message channel, there can be multiple `@ErrorListener` methods consuming the error messages and process them based on the requirements.
+
+[NOTE]
+====
+
+Since the error messages received from the `errorChannel` is always of the type `ErrorMessage`, the `@ErrorListener` annotated method should have parameter argument type `ErrorMessage`.
```

Comment:
`are always` . Because that was `messages` - plural.

Why is it always `ErrorMessage`? Why can't I rely on the `Exception` `payload`? Or be more strict and expect `MessagingException` to get an access to the `failedMessage` without any casts.

Thanks